core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackService.java

```diff
@@ -214,11 +214,11 @@ public Long countAliveByEnvironment(Environment environment) {
         }
     }
 
-    public StackResponse findStackByNameAndWorkspaceId(String name, Long workspaceId) {
+    public Stack findStackByNameAndWorkspaceId(String name, Long workspaceId) {
         try {
             return transactionService.required(() ->
-                    conversionService.convert(stackRepository
-                            .findByNameAndWorkspaceIdWithLists(name, workspaceId), StackResponse.class));
+                    stackRepository
+                            .findByNameAndWorkspaceIdWithLists(name, workspaceId));
```

Comment:
I think It will produce LazyInitException or other cases the transaction is unnecessary.

Reply:
No, it won't cause lazy. You are right the transaction is not necessary here, since it contains only a single repo call and repositories are opening a transaction anyway

Reply:
If the converter references to a lazy loaded field of the stack, then there will be a lazy init exception without a transaction. This why we had some lazy exceptions in environment related stuff.

Reply:
Lazy cannot occur here, except if you access lazy loaded field of Stack, but wrapping the repository again will not do too much difference. Resolving it now.